### PR TITLE
update tool_schema for demos and specify timestamps for demo 3

### PR DESCRIPTION
**Pull request by linhktran**
In this PR, I have added 3 demo scenarios for the use of LLMCAM. For most of the part, it looks promising. However, there are 2 things to notice (at least when I was testing): 
1. According to GPT, it cannot retroactively access past moments from Live video (we have to tell it to capture first)
2. For the parking lots demo, I need to tell GPT at which time we want to check the availablity, and the result returned by GPT is not really correct (it says 5, but there are a lot more).

I have also exported a small part in 10_store.ipynb so that store.py can be imported.

### CODE REVIEW

File: llmcam/store.py [hunk 1]
```diff
@@ -61,6 +61,10 @@ def load_oas(
     overwrite=False
 )
 
+# %% ../nbs/10_store.ipynb 11
+from .fn_to_fc import complete, form_msgs, print_msgs
+from .oas_to_requests import generate_request
```
Comment (doyu) [inline llmcam/store.py:66]: https://chatgpt.com/share/673eba22-2014-8002-a316-c91383f783ca

should we move onto `from .fn_to_fc import *` ?

**Review by doyu — APPROVED** [review]
Please merge!

File: nbs/10_store.ipynb
Comment (doyu) [inline nbs/10_store.ipynb:1]: Remove this for now.

**Review by doyu — CHANGES_REQUESTED** [review]

**Comment by doyu** [comment]
> According to GPT, it cannot retroactively access past moments from Live video (we have to tell it to capture first)

Yes, that's why file manager is needed to retreive the past ones.

> For the parking lots demo, I need to tell GPT at which time we want to check the availablity, and the result returned by GPT is not really correct (it says 5, but there are a lot more).

Can this be escalated into YOLO FC?

**Comment by linhktran** [comment]
I have changed the import in the 10_store.ipynb and move it to the top so that we only need to do it once. 
For the parking lots, AFAIK, YOLO doesn't have such available annotations, but we need to manual do it ourselves as stated in this webpage https://docs.ultralytics.com/guides/parking-management/.

**Pull request merged**